apps/homescreen/js/app.js

```diff
@@ -523,6 +527,14 @@ const SETTINGS_VERSION = 0;
 
       // Display the top shadow when scrolling down
       case 'scroll':
+        if (e.target === this.panels) {
```

Comment:
Is this for updating the horizontal pages indicator? In that case, do we have any event when that transition finishes, instead of checking it on every sroll?